spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/KubernetesClientSecretsPropertySource.java

```diff
@@ -40,13 +40,14 @@
 	private CoreV1Api coreV1Api;
 
 	public KubernetesClientSecretsPropertySource(CoreV1Api coreV1Api, String name, String namespace,
-			Environment environment, Map<String, String> labels) {
-		super(getSourceName(name, namespace), getSourceData(coreV1Api, environment, name, namespace, labels));
+			Environment environment, Map<String, String> labels, boolean useNameAsPrefix) {
+		super(getSourceName(name, namespace),
+				getSourceData(coreV1Api, environment, name, namespace, labels, useNameAsPrefix));
 
 	}
 
 	private static Map<String, Object> getSourceData(CoreV1Api api, Environment env, String name, String namespace,
-			Map<String, String> labels) {
+			Map<String, String> labels, boolean useNameAsPrefix) {
```

Comment:
Currently, I have implemented the solution only for secrets. However, I initially created the issue for secrets and configmaps.  

If we also want to implement this for configmaps, I think it would be great if the config parameters would be the same.

Reply:
ah! good point.